src/UI/Component/Chart/Gauge/Standard.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/* Copyright (c) 2017 Ralph Dittrich <dittrich@qualitus.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Chart\Gauge;
+
+/**
+ * Interface Standard
+ * @package ILIAS\UI\Component\Chart\Gauge
+ */
+interface Standard extends Gauge {
+
+    /**
+     * Get comparision value
+     *
+     * This value is represented as the second gauge bar.
+     *
+     * @return int|float
+     */
+    public function getComparision();
+
+    /**
+     * Get clone of Gauge with main text
+     *
+     * It will be displayed above the main value percentage display.
+     * Example: withMainText('Your Score')
+     *
+     * @param string $text
+     * @return \ILIAS\UI\Component\Chart\Gauge\Gauge
+     */
+    public function withMainText($text);
+
+    /**
+     * Get main text value
+     *
+     * @return string

```

Comment:
This could be `null` as well, or am I missing something?

Reply:
I had to check this again and you are right.